aws_etl_tools/redshift_ingest/ingestors.py

```diff
@@ -61,24 +64,38 @@ def _ingest_query(self):
                 upsert_match_statement=self._upsert_match_statement()
             )
 
+    @property
+    def default_copy_commands(self):
+        return ["EMPTYASNULL", "BLANKSASNULL", "TIMEFORMAT AS 'auto'", "STATUPDATE ON"]
+
+    @property
+    def csv_copy_commands(self):
+        return ["CSV", "IGNOREBLANKLINES"]
+
     def _copy_statement(self):
-        return """
+        copy_commands = self.default_copy_commands
+        copy_commands.append('MANIFEST') if self.manifest else None
+        copy_commands.append('GZIP') if self.gzip else None
+
+        if self.jsonpaths:
+            copy_commands.append("JSON '{}'".format(self.jsonpaths))
+        else:
+            copy_commands.extend(self.csv_copy_commands)
```

Comment:
what about a single property called `_copy_parameters` (which is the wording aws uses in the docs) to handle all of this branching logic and defaults? if we need more complex logic, we can always peel off an object to handle it, but at least putting it all in one place will force us to see everything that's going on together.